src/member/migrations/0027_auto_20170221_1438.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-21 14:38
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('member', '0026_change_to_emergency_contacts'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='emergencycontact',
+            unique_together=set([('client', 'member')]),
+        ),
+    ]
```

Comment:
I agree, in production the migration 0026 is not applied, is better to merge both migrations.